README.md

```diff
@@ -3,13 +3,25 @@
 [![Build Status: Ubuntu Bionic (Actions)](https://github.com/ros-industrial/universal_robot/workflows/CI%20-%20Ubuntu%20Bionic/badge.svg?branch=melodic-devel)](https://github.com/ros-industrial/universal_robot/actions?query=workflow%3A%22CI+-+Ubuntu+Bionic%22)
 [![Build Status: Ubuntu Focal (Actions)](https://github.com/ros-industrial/universal_robot/workflows/CI%20-%20Ubuntu%20Focal/badge.svg?branch=melodic-devel)](https://github.com/ros-industrial/universal_robot/actions?query=workflow%3A%22CI+-+Ubuntu+Focal%22)
 
-[![license - apache 2.0](https://img.shields.io/:license-Apache%202.0-yellowgreen.svg)](https://opensource.org/licenses/Apache-2.0)
+[![License - apache 2.0](https://img.shields.io/:license-Apache%202.0-yellowgreen.svg)](https://opensource.org/licenses/Apache-2.0)
 [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+[![License](https://img.shields.io/badge/License-Universal%20Robots%20A/S%E2%80%99%20Terms%20and%20Conditions%20for%20Use%20of%20Graphical%20Documentation-blue.svg)](https://www.universal-robots.com/legal/terms-and-conditions/terms_and_conditions_for_use_of_graphical_documentation.txt)
```

Comment:
Should we instead make this point to the `ur_description/meshes/ur20/LICENSE.txt` file instead? The online version could be changed at any time. The files in this repository would fall under the version of the license as it is right now, in this PR. It would make sense to me to make this badge link to the version the files would fall under.

Reply:
From what I understood, the license is structured such that the latest version always is relevant (see point 1.4)

> ... If you download a software package that includes the Graphical Documentation, your use of the Graphical Documentation will be subject to and governed by the latest version of the T&Cs, which can be found here: ...

Given that clause I think it does make sense to link to the online version.

Reply:
I'm not sure that's legal. There would be no change notification to current users, UR would be able to silently, unilaterally change the complete license, and with all licenses I've worked with, once you have a copy of the artefacts under that license, that copy and/or version will always fall under the version of the license as it was when you agreed to it.

There is already a copy of the current version in the repository -- or at least there will be, after merging this PR.

It's best practice to include a copy of the license in all cases, even with OSS licenses. We don't do that currently in this repository, but that's an omission.

Reply:
The same section 1.4 also states:
> However, you may only make the Graphical Documentation or any whole or partial copies thereof available to the public via download if it is part of a downloadable software package that includes a copy of the T&Cs.

which is what this PR will result in: a copy will be included.

So it doesn't seem strange to me to then point to that copy, and not the online version.

That seems like it would just lead to confusing situations.

Reply:
You have a point. Though the motivation was to remove the burden from the repo owner to ensure that the license is up to date.